spec/controllers/coronavirus-form/offer_community_support_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe CoronavirusForm::OfferCommunitySupportController, type: :controller do
+  let(:current_template) { "coronavirus_form/offer_community_support" }
+  let(:session_key) { :offer_community_support }
+
+  describe "GET show" do
+    it "renders the form" do
+      get :show
+      expect(response).to render_template(current_template)
+    end
+  end
+
+  describe "POST submit" do
+    let(:selected_yes) { I18n.t("coronavirus_form.offer_care.options.option_yes.label") }
+    let(:selected_no) { I18n.t("coronavirus_form.offer_care.options.option_no.label") }
```

Comment:
It looks like `selected_no` isn't used here

Reply:
this is true! I could remove it, will immediately add a test for both yes and no, when i merge q9

Reply:
and that test will use :selected_no